debezium-connector-mysql/pom.xml

```diff
@@ -411,6 +430,7 @@
                                 <database.replica.password>${mysql.replica.password}</database.replica.password>
                                 <database.port>${mysql.port}</database.port>
                                 <database.replica.port>${mysql.port}</database.replica.port>
+				<test.mysql.gtid>${test.mysql.gtid}</test.mysql.gtid>
```

Comment:
Would be a good idea to have it ensured by CheckStyle during the build. I've filed [DBZ-275](https://issues.jboss.org/browse/DBZ-275) for tracking it.